test/test_transforms_v2.py

```diff
@@ -418,130 +5621,128 @@ def test_detection_preset(image_type, data_augmentation, to_tensor, sanitize):
     assert out["boxes"].shape[0] == out["masks"].shape[0] == out["label"].shape[0] == num_boxes_expected
 
 
-@pytest.mark.parametrize("min_size", (1, 10))
-@pytest.mark.parametrize("labels_getter", ("default", lambda inputs: inputs["labels"], None, lambda inputs: None))
-@pytest.mark.parametrize("sample_type", (tuple, dict))
-def test_sanitize_bounding_boxes(min_size, labels_getter, sample_type):
-
-    if sample_type is tuple and not isinstance(labels_getter, str):
-        # The "lambda inputs: inputs["labels"]" labels_getter used in this test
-        # doesn't work if the input is a tuple.
-        return
-
-    H, W = 256, 128
-
-    boxes_and_validity = [
-        ([0, 1, 10, 1], False),  # Y1 == Y2
-        ([0, 1, 0, 20], False),  # X1 == X2
-        ([0, 0, min_size - 1, 10], False),  # H < min_size
-        ([0, 0, 10, min_size - 1], False),  # W < min_size
-        ([0, 0, 10, H + 1], False),  # Y2 > H
-        ([0, 0, W + 1, 10], False),  # X2 > W
-        ([-1, 1, 10, 20], False),  # any < 0
-        ([0, 0, -1, 20], False),  # any < 0
-        ([0, 0, -10, -1], False),  # any < 0
-        ([0, 0, min_size, 10], True),  # H < min_size
-        ([0, 0, 10, min_size], True),  # W < min_size
-        ([0, 0, W, H], True),  # TODO: Is that actually OK?? Should it be -1?
-        ([1, 1, 30, 20], True),
-        ([0, 0, 10, 10], True),
-        ([1, 1, 30, 20], True),
-    ]
+class TestSanitizeBoundingBoxes:
```

Comment:
The only change here is that I merged three individual test functions into one class. The tests itself are copy-pasted.